include/gx_layers.h

```diff
@@ -44,9 +44,9 @@ typedef struct GXBanksConfig {
     GXVRamSubOBJExtPltt subobjextpltt;
     GXVRamTex tex;
     GXVRamTexPltt texpltt;
-} GF_GXBanksConfig;
+} GraphicsBanks;
```

Comment:
Shouldn't the typedef name match the struct name?

Reply:
oh whoops, forgot to change that